packages/cli/test/integration/passwordReset.endpoints.test.ts

```diff
@@ -0,0 +1,231 @@
+import express = require('express');
+import { getConnection } from 'typeorm';
+import { v4 as uuid } from 'uuid';
+
+import * as utils from './shared/utils';
+import { Db } from '../../src';
+import config = require('../../config');
+import { compare } from 'bcryptjs';
+
+let app: express.Application;
+
+beforeAll(async () => {
+	app = utils.initTestServer({ namespaces: ['passwordReset'], applyAuth: true });
+	await utils.initTestDb();
+	await utils.truncateUserTable();
+});
+
+beforeEach(async () => {
+	jest.isolateModules(() => {
+		jest.mock('../../config');
+	});
+
+	config.set('userManagement.hasOwner', true);
+	config.set('userManagement.emails.mode', '');
+
+	const globalOwnerRole = await Db.collections.Role!.findOneOrFail({
+		name: 'owner',
+		scope: 'global',
+	});
```

Comment:
also here: maybe move to global `let` to not fetch it for every test

and create a getter in the `shared`